pkgs/development/tools/git-ps-rs/default.nix

```diff
@@ -3,38 +3,29 @@
 , rustPlatform
 , stdenv
 , pkg-config
-, libgpg-error
-, gpgme
 , dbus
 , openssl
 , Security
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "git-ps-rs";
-  version = "6.9.0";
+  version = "7.0.0";
 
   src = fetchFromGitHub {
     owner = "uptech";
     repo = "git-ps-rs";
     rev = version;
-    hash = "sha256-D6613T87jLEur8WXHed2cSKVafKVfgGWap/z/UBe31U=";
+    hash = "sha256-HPHFjYfug642NXeNmv50UzsdOAlDR9F/MtgYnzwiZP8=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
```

Comment:
lockfile can be removed and replanced by cargoHash

Reply:
Removed lockfile thanks!